src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,95 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int MAX_NUMBER = 10;
```

Comment:
```suggestion
    private static final int INITIAL_SIZE = 10;
```
 з макс намбером можна подумати, що ліст макс 10 елементів вміщає